Exiled.API/Features/Player.cs

```diff
@@ -389,26 +385,25 @@ public string CustomInfo
                         if (token.StartsWith("color=", StringComparison.Ordinal))
                         {
                             if (token.Length < 14 || token[13] != '>')
-                                Log.Error($"Custom info of player {Nickname} has been REJECTED. reason: (Bad text reject) Info: {value}");
+                                Log.Error($"Custom info of player {Nickname} has been REJECTED. \nreason: (Bad text reject) \ntoken: {token} \nInfo: {value}");
                             else if (!Misc.AllowedColors.ContainsValue(token.Substring(6, 7)))
-                                Log.Error($"Custom info of player {Nickname} has been REJECTED. reason: (Bad colour reject) Info: {value}");
+                                Log.Error($"Custom info of player {Nickname} has been REJECTED. \nreason: (Bad colour reject) \ntoken: {token} \nInfo: {value}");
                         }
                         else if (token.StartsWith("#", StringComparison.Ordinal))
                         {
                             if (token.Length < 8 || token[7] != '>')
-                                Log.Error($"Custom info of player {Nickname} has been REJECTED. reason: (Bad text reject) Info: {value}");
+                                Log.Error($"Custom info of player {Nickname} has been REJECTED. \nreason: (Bad text reject) \ntoken: {token} \nInfo: {value}");
                             else if (!Misc.AllowedColors.ContainsValue(token.Substring(0, 7)))
-                                Log.Error($"Custom info of player {Nickname} has been REJECTED. reason: (Bad colour reject) Info: {value}");
+                                Log.Error($"Custom info of player {Nickname} has been REJECTED. \nreason: (Bad colour reject) \ntoken: {token} \nInfo: {value}");
                         }
                         else
                         {
-                            Log.Error($"Custom info of player {Nickname} has been REJECTED. reason: (Bad text reject) Info: {value}");
+                            Log.Error($"Custom info of player {Nickname} has been REJECTED. \nreason: (Bad colour reject) \ntoken: {token} \nInfo: {value}");
```

Comment:
Colour?

Reply:
> Colour?

That's British English Lol. And yes, use US English.

Reply:
sorry we learned Britsh english at school lol